MediaModelBundle/Repository/MediaRepository.php

```diff
@@ -51,7 +51,12 @@ public function findByKeywords($keywords)
     {
         $qb = $this->createQueryBuilder();
 
-        $qb->field('keywords.label')->in(explode(',', $keywords));
+        $keywords = explode(',', $keywords);
+        foreach($keywords as $key => $keyword) {
+            $keywords[$key] = new \MongoId($keyword);
```

Comment:
I'm not sure about this: if we want to get the mongo ids of existing keyword documents, we should then search for them in the database, not regenerate new ones ?